es2csv.py

```diff
@@ -234,7 +234,7 @@ def write_to_csv(self):
                     timer += 1
                     bar.update(timer)
                     line_as_dict = json.loads(line)
-                    line_dict_utf8 = {k: v.encode('utf8') if isinstance(v, str) else v for k, v in line_as_dict.items()}
+                    line_dict_utf8 = {k: v if isinstance(v, str) else v for k, v in line_as_dict.items()}
```

Comment:
Are we going to lose python2.7 support after this? Will unicode symbols be parsed correctly and wrote to file?